src/ast/mod.rs

```diff
@@ -1759,6 +1759,11 @@ pub enum Statement {
         replace_into: bool,
         /// Only for mysql
         priority: Option<MysqlInsertPriority>,
+        /// Only for mysql
+        #[cfg_attr(feature = "visitor", visit(with = "visit_relation"))]
+        row_alias: ObjectName,
```

Comment:
I think the typical pattern is to use `Option<ObjectName>` for optional fields. 

Perhaps the issue here is that we can only use `visit_relation` with a `ObjectName` 🤔 

In that case we could make a new structure like

```
struct InsertAliases { 
  row_alias: ObjectName,
  col_aliases: Option<Vec<Ident>>,
}
```

Or maybe just avoid the `visitor` call 🤔  (and could file a ticket with a bug enhancment)

Reply:
The issue was visit_relation with a ObjectName. I think, InsertAliases struct is best solution. I implemented the solution.